<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>331</y>
      <width>47</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>261</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEditIP">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>IP do servidor</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButtonDisconnect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>101</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Timing(s)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="horizontalSliderTime">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSlider" name="horizontalSliderMin">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="lcdNumberMax"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="lcdNumberTime">
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="lcdNumberMin"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QSlider" name="horizontalSliderMax">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>361</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>331</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconectado</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTime</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberTime</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
